### PR TITLE
HOC via hooks for ac/fv

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

File: assets/src/components/FileAccessChart.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-  export default FileAccessChart
\ No newline at end of file
+import createFileAccessChart from './d3/createFileAccessChart'
+import React, { useState } from 'react'
+import useResponsiveness from '../hooks/useResponsiveness'
+import useCreateChart from '../hooks/useCreateChart'
+
+function FileAccessChart (props) {
+  const [domElement, setDomElement] = useState(null)
+  console.log(props)
```
Comment (justin0022) [inline assets/src/components/FileAccessChart.js]: `console.log` can be removed

File: assets/src/components/HorizontalBar.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
 import createHorizontalBar from './d3/createHorizontalBar'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
+import { useState } from 'react'
+import useCreateChart from '../hooks/useCreateChart'
+import React from 'react'
```
Comment (justin0022) [inline assets/src/components/HorizontalBar.js]: Lines 2 and 4 should be combined into 
`import React, { useState } from 'react`

File: assets/src/components/FileAccessChart.js (hunk 1 above)
Comment (pushyamig) [reply assets/src/components/FileAccessChart.js]: Done

File: assets/src/components/HorizontalBar.js (hunk 2 above)
Comment (pushyamig) [reply assets/src/components/HorizontalBar.js]: Done

**Pull request merged**